CourseApp.Tests/Sum5in2Re3.cs

```diff
@@ -0,0 +1,19 @@
+namespace TestsSum
+{
+    using CourseApp;
+    using Xunit;
+
+    public class Sum_5in2_Re3
```

Comment:
tests for single method shoud be placed in 1 file in several methods 

just create CalculatorTests.cs (Or even Calculator.cs) in test package and place all code there